README.md

```diff
@@ -2,12 +2,57 @@
 
 ## Learning Goals
 
-1. SWBAT 1
+1. What are JavaScript methods
 
 ## Introduction
 
-## SWBAT 1
+In the previous lesson, we did a very important thing in the JavaScript
+language, we _called a method_. We glossed some of the detail about what
+was happening at the technical level here. Let's put some technical
```

Comment:
Let's replace "technical vocabulary" with "new vocabulary" to both soften it and avoid repeating the word "technical."